# Awesome(?) somatic variant callers
A collection of somatic variant callers. Awesomeness to be reviewed üßê

- [ ] [gatk Mutect2](https://gatkforums.broadinstitute.org/gatk/discussion/24057/how-to-call-somatic-mutations-using-gatk4-mutect2#latest)
- [ ] [ISOWN](https://github.com/ikalatskaya/ISOWN): accurate somatic mutation identification in the absence of normal tissue controls.
- [ ] [LoFreq version 2](https://csb5.github.io/lofreq/): Fast and sensitive variant calling from next-gen sequencing data
- [ ] [SMuRF](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz018/5288515): portable and accurate ensemble prediction of somatic mutations
- [ ] [OHVarfinDer](https://github.com/takumorizo/OHVarfinDerstars)
- [ ] [PipeIT](https://jmd.amjpathol.org/article/S1525-1578(18)30338-6/fulltext): A Singularity Container for Molecular Diagnostic Somatic Variant Calling on the Ion Torrent Next-Generation Sequencing Platform
- [ ] [NeoMutate](https://link.sprnger.com/article/10.1186/s12920-019-0508-5): an ensemble machine learning framework for the prediction of somatic mutations in cancer


# Reviews

- [ ] [A review of somatic single nucleotide variant calling algorithms for next-generation sequencing data](https://www.semanticscholar.org/paper/A-review-of-somatic-single-nucleotide-variant-for-Xu/81b4246c90c3c8036ff719b9d4c3d5e83fbb32dc)


# BONUS
Data caveats and attributes to be cautious of
- [ ] tumor cellularity 
